File: Tourtlee.BookingSystem/Tourtlee.BookingSystem.Business/Dto/Accounts/CreateUserDto.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace Tourtlee.BookingSystem.Business.Dto.Accounts
{
    public class CreateUserDto
    {
        [Required, DataType(DataType.EmailAddress)]
        public string Email { get; set; }

        [Required, DataType(DataType.Password)]
        public string Password { get; set; }

        [Required]
        public string PasswordRepeat { get; set; }

        public Guid? IdOrganization { get; set; }
        public string OrganizationName { get; set; }
        
        public CreateUserOrganizatioModes OrganizationMode { get; set; }
    }

    public enum CreateUserOrganizatioModes
    {
        Existing = 0,
        Create = 1
    }
}

File: Tourtlee.BookingSystem/Tourtlee.BookingSystem.Web/wwwroot/app/admin/users/dto/CreateUserDto.js
var CreateUserDto = (function () {
    function CreateUserDto() {
    }
    return CreateUserDto;
})();
exports.CreateUserDto = CreateUserDto;
(function (CreateUserOrganizationModes) {
    CreateUserOrganizationModes[CreateUserOrganizationModes["Existing"] = 0] = "Existing";
    CreateUserOrganizationModes[CreateUserOrganizationModes["Create"] = 1] = "Create";
})(exports.CreateUserOrganizationModes || (exports.CreateUserOrganizationModes = {}));
var CreateUserOrganizationModes = exports.CreateUserOrganizationModes;
